docs/tutorials/projects/tipsy.md

```diff
@@ -333,10 +329,8 @@ Click on the **Services** subtab.
 
    ![mlmodel service panel with empty sections for Model Path, and Optional Settings such as Label Path and Number of threads.](/tutorials/tipsy/app-service-ml-before.png)
 
-   Select the **Path to existing model on robot** for the **Deployment** field.
-   Then specify the absolute **Model path** as <file>/home/tipsy/effdet0.tflite</file> and any **Optional settings** such as the absolute **Label path** as <file>/home/tipsy/labels.txt</file> and the **Number of threads** as 1.
-
-   ![mlmodel service panel, Deployment selected as Path to Existing Model On Robot, Model Path filled as /home/tipsy/effdet0.tflite and Label Path filled as /home/tipsy/labels.txt, Number of threads is 1.](/tutorials/tipsy/app-service-ml-after.png)
+   Select the **Deploy model on robot** for the **Deployment** field.
+   Then select the `bill:effdet0` model from the **Models** dropdown.
```

Comment:
No-op: Future concern: who is Bill, why am I being billed for use of this model, I only trust Viam namespace models per last sentence of [this section](https://docs.viam.com/registry/#naming-your-model-namespacerepo-namename) for my production healthcare heart replacement robot, etc.

Reply:
Yeah I thought that too we should probably flag that